db/seeds.rb

```diff
@@ -44,12 +44,14 @@ def build_menus_with_dishes(menu_names)
 
 def add_dish(number, menu, restaurant)
   price = rand(20..60)
-  prefix = restaurant.cuisine
+  prefix = restaurant.category
   Dish.create(name: "#{prefix} Dish #{number}",
               price: price,
-              description: "A great #{restaurant.cuisine} dish!",
-              menu: menu,
-              category: Dish::CATEGORIES.sample)
+              description: "A great #{restaurant.category} dish!",
+              # category: Dish::CATEGORIES.sample
```

Comment:
Each dish should have a category. Even if we are skipping validations in this scrip we will have issues when we will display the dishes on the meny#show page (considering that Viktoria and Jesper will group the display in their upcoming PR)